src/components/ConcentrationRisk/ConcentrationRisk.js

```diff
@@ -138,15 +138,13 @@ class ConcentrationRisk extends PureComponent {
       showContent = (
         <Loading title='concentrationrisk.title' />
       )
-    } else if (!chartData.length) {
+    } else if (!entries.length) {
       showContent = (
         <Fragment>
           <h4>
             {t('concentrationrisk.title')}
             &nbsp;
```

Comment:
I know this is not related, but let's use {' '} for space.

Reply:
@jarindr got it https://github.com/bitfinexcom/bfx-report-ui/pull/249 :)

Reply:
1 PR to kill them all 😆